substrate/bin/node/runtime/src/lib.rs

```diff
@@ -177,7 +177,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 268,
-	impl_version: 0,
+	impl_version: 1,
```

Comment:
```suggestion
	impl_version: 0,
```

The kitchensink runtime needs to bumping.

Reply:
> The kitchensink runtime needs to bumping.

Assuming 'no' bumping was meant - is this because its also bumped automatically on release like the spec_version or because the change to the runtime was sufficient to warrant a bump?